docs/usage/private-modules.md

```diff
@@ -64,7 +64,7 @@ It can be either a "platform" name (e.g. `github`, `azure`, etc) or a "datasourc
 `baseUrl` can be used if you want to only apply the credentials for a nested path within the host, e.g. `https://registry.company.com/nested/path/`.
 If the same credentials apply to all paths on a host, then use `hostName` instead, e.g. `registry.company.com`.
 Finally, to apply credentials to all hosts within the domain, use `domainName`, e.g. `company.com`.
-You need to pick only one of these and not try to use multiple at the same time, or it will be a config error.
+You need to pick _only one_ of these and not try to use multiple at the same time, or you will end up with configuration errors.
```

Comment:
```suggestion
You need to pick _only one_ of these and not configure more than one of these fields within the same host rule, otherwise it will error.
```